tests/functional/monitoring/prometheus/alerts/test_alert_mds_cpu_high_usage.py

```diff
@@ -0,0 +1,102 @@
+import time
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import tier2
+from ocs_ci.ocs import constants
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import cluster
+from ocs_ci.utility import prometheus
+from ocs_ci.framework.pytest_customization.marks import magenta_squad
+from ocs_ci.utility.utils import ceph_health_check_base
+
+log = logging.getLogger(__name__)
+
+
```

Comment:
we miss tier, polarion-id, and squad decorators

Reply:
Added tier and squad name just above the class. Polarion id added for all tests.